yaksh/views.py

```diff
@@ -69,6 +70,13 @@ def has_profile(user):
     """ check if user has profile """
     return True if hasattr(user, 'profile') else False
 
+def add_teachers_to_group(teachers):
```

Comment:
instead you can name it as add_to_group
And instead of teachers as parameter, you can say users.